File: back/app/model/MemoData.py
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.schema import Column
from sqlalchemy.types import Integer, String, DateTime
Base=declarative_base()
class MemoData(Base):
    __tablename__="memoDataTable" #テーブル名を指定
    user=Column(Integer, primary_key=True)
    id=Column(Integer, primary_key=True)
    title=Column(String(255))
    body=Column(String(255))
    datetime = Column(DateTime())

    def toDict(self):
        return {
            "id":self.id,
            "title":self.title,
            "body":self.body,
            "datetime":str(self.datetime)
            }

File: front/src/MemoData.ts
class MemoData {
  user: string;
  id: number;
  private _title: string;
  public get title(): string {
    return this._title;
  }
  public set title(value: string) {
    this._title = value;
  }
  private _body: string;
  public get body(): string {
    return this._body;
  }
  public set body(value: string) {
    this._body = value;
  }
  datetime: string;
  constructor() {
    this.user = "";
    this.id = 0;
    this._title = ""
    this._body = "";
    this.datetime = "";
  }
}

export default MemoData;